### DISCUSSION TITLE
How do I add a bookmark?

### DISCUSSION THREAD

**Discussion by gewaleelek** [Slion/Fulguris#651] [Q&A]
Hello, I'm a completely new user!

I'm sure I'm overlooking some obvious option somewhere in the app, but I've been trying to add a page to the bookmarks to no avail. 

Perhaps my query wasn't good enough, but a quick search on the GitHub issue page yielded some results along the line of "adding bookmarks are hard to find" and "bookmarks need revamping" but not how to do it. 

Could you please help point me out:
* How to add a page to the bookmarks and
* How to create a folder in the bookmarks? 

Thank you very much!

**Comment by Slion** [comment]
Menu > Web page > Add bookmark

For folder creation just specify the folder in the bookmark creation dialog.

**Comment by FrankwaP** [comment]
> I'm sure I'm overlooking some obvious option somewhere in the app, but I've been trying to add a page to the bookmarks to no avail.

Yes I did not get that pressing "Web Page" would provide tools like bookmarks or text search.  
I understood it as "displaying the web version of the page" (instead of mobile version). Maybe "Web Page Tools" would be more explicit?

Great app BTW, I have an +10y phone and it runs very smoothly. Merci beaucoup Stéphane :-)

**Comment by Slion** [comment]
> Yes I did not get that pressing "Web Page" would provide tools like bookmarks or text search.
I understood it as "displaying the web version of the page" (instead of mobile version). Maybe "Web Page Tools" would be more explicit?

Good point, maybe it should just say "More" with the menu icon. I guess there could be a chevron on the right too, to indicate this is a sub-menu.
